webpack.wordpress.config.js

```diff
@@ -1,12 +1,16 @@
 /** @format */
 const path = require( 'path' );
 const webpack = require( 'webpack' );
+const UglifyJsPlugin = require( 'uglifyjs-webpack-plugin' );
 const LodashModuleReplacementPlugin = require( 'lodash-webpack-plugin' );
 
-module.exports = {
+const env = process.env.NODE_ENV;
+
+const config = {
```

Comment:
Not sure if I should mention this here as I've already mentioned it in the first PR: I'd prefer that targets were independent of each other. We should have a `targets/wordpress/index.js` independent from `targets/browser/index.js`.

Reply:
Yes you did, and this was fixed in the base PR.